docs/integration-services/packages/dtexec-utility.md

```diff
@@ -133,7 +135,14 @@ EXEC @returncode = xp_cmdshell 'dtexec /f "C:\UpsertData.dtsx"'
 ```  
   
 > **IMPORTANT!!** In [!INCLUDE[msCoName](../../includes/msconame-md.md)][!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], the **xp_cmdshell** option is disabled by default on new installations. The option can be enabled by running the **sp_configure** system stored procedure. For more information, see [xp_cmdshell Server Configuration Option](../../database-engine/configure-windows/xp-cmdshell-server-configuration-option.md).  
-  
+
+##  <a name="bash"></a> Using dtexec from Bash
+**Using dtexec from Bash**

```

Comment:
@jeroenterheerdt  Strike line 140, since line 139 will be the same text already bold from the H2 `##`

Reply:
done, but notice it is the same on this page in a number of places, see line 121 / 122.